src/components/Jobs/components/PreviewsJobs/PreviewsJobs.js

```diff
@@ -1,39 +1,38 @@
 import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-import get from 'lodash/get';
+import { FormattedMessage } from 'react-intl';
+import {
+  get,
+  filter,
+} from 'lodash';
 
 import JobsList from '../JobsList';
 import sortPreviewJobs from './sortPreviewJobs';
 import { DataFetcherContext } from '../../../DataFetcher/DataFetcherContext';
+import {
+  READY_FOR_PREVIEW,
+  PREPARING_FOR_PREVIEW,
+} from '../../jobStatuses';
 
 class PreviewsJobs extends PureComponent {
-  static propTypes = {
-    checkDateIsToday: PropTypes.func.isRequired,
-    noJobsMessage: PropTypes.node,
-  };
+  static contextType = DataFetcherContext;
 
-  static defaultProps = {
-    noJobsMessage: '',
-  };
+  prepareJobsData() {
+    let jobs = get(this.context, ['jobs', 'itemsObject', 'jobExecutions'], []);
 
-  static contextType = DataFetcherContext;
+    jobs = filter(jobs, ({ status }) => status === READY_FOR_PREVIEW || status === PREPARING_FOR_PREVIEW);
```

Comment:
Let's extract needed statuses to the instance variable and then use there in the 
`jobStatuses = [READY_FOR_PREVIEW, PREPARING_FOR_PREVIEW]`
and use there like below
`jobStatuses.includes(status)`
so we won't need to extend the `if` condition every time when the new status may arrive.